### ISSUE TITLE
decode-stacktrace.py doesn't work on Arch Linux

### ISSUE THREAD

**Issue by max-weller** [SmingHub/Sming#1227]
...because on Arch the default python version is 3.x, and decode-stacktrace.py requires 2.x, however the shebang line in the script references the unversioned python binary.

This can be fixed by PR #1225.

**Comment by slaff** [comment]
@max-weller decode-stacktrace should work also with python3. Please, paste here the issues that you have. A proper fix IMHO would be making the script work also with python3 and not force it to use always python2.

**Comment by max-weller** [comment]
That'd be the clean solution I guess :D

**Comment by max-weller** [comment]
I get the following error after pasting a stack trace:
```
Traceback (most recent call last):
  File "tools/decode-stacktrace.py", line 50, in <module>
    pipe.stdin.write(line)
TypeError: a bytes-like object is required, not 'str'
```

**Comment by max-weller** [comment]
OK it works now with Python3, I made a new PR  #1228

**Comment by slaff** [comment]
Should be fixed with PR rebase #1229.
